### DISCUSSION TITLE
Support for arrow keys to navigate flame graph displayed in webview

### DISCUSSION THREAD

**Discussion by manav-bhanot** [microsoft/vscode-discussions#1198] [Extension Development QnA]
I have a VSCode extension that displays a flame graph in a Webview.  [d3-flame-graph](https://github.com/spiermar/d3-flame-graph) lib is used to display the flame graph.
 
Are there any recommended ways to navigate the flame graph using arrow keys on the keyboard, similar to how a user would navigate a flame graph in chrome dev tools using arrow keys on keyboard ?

**Comment by isidorn** [comment]
Please file that as a feature request against that extension. In theory they could support this
